### PR TITLE
Pull oraclelinux image from ghcr.io so that customers aren't rate limited

### PR DESCRIPTION
**Pull request by rjeberhard**

### CODE REVIEW

File: imagetool/src/main/java/com/oracle/weblogic/imagetool/util/DockerfileOptions.java [hunk 1]
```diff
@@ -91,7 +91,7 @@ public DockerfileOptions(String buildId) {
 
         tempDirectory = "/tmp/imagetool";
 
-        baseImageName = "oraclelinux:7-slim";
+        baseImageName = "ghcr.io/oracle/oraclelinux:7-slim";
```
Comment (ddsharpe) [inline imagetool/src/main/java/com/oracle/weblogic/imagetool/util/DockerfileOptions.java:94]: Did you mean for this to be `8-slim` as well?

File: tests/src/test/java/com/oracle/weblogic/imagetool/tests/ITImagetool.java [hunk 2]
```diff
@@ -965,7 +965,7 @@ void createMiiOl8slim(TestInfo testInfo) throws Exception {
             String tagName = build_tag + ":" + getMethodName(testInfo);
             String command = new CreateCommand()
                 .tag(tagName)
-                .fromImage(" oraclelinux", "8-slim")
+                .fromImage(" ghcr.io/oracle/oraclelinux", "8-slim")
```
Comment (ddsharpe) [inline tests/src/test/java/com/oracle/weblogic/imagetool/tests/ITImagetool.java:968]: I think this leading space should be removed.

Comment (rjeberhard) [reply tests/src/test/java/com/oracle/weblogic/imagetool/tests/ITImagetool.java:968]: Happily, I kept it because it was there before :)

**Comment by rjeberhard** [comment]
@ddsharpe, okay, I think that this is ready.

**Review by ddsharpe — APPROVED** [review]

**Review by jshum2479 — APPROVED** [review]

**Pull request merged**